OPHD/States/MapViewState.cpp

```diff
@@ -741,7 +741,7 @@ void MapViewState::onChangeDepth(int oldDepth, int newDepth) {
 
 void MapViewState::moveView(MapOffset offset) {
 	int oldZLevel = mMapView->currentDepth();
-	
+
```

Comment:
I approve of whitespace fixes. Preferably they should be in a separate commit from code changes.